src/Randomizer.SMZ3.Tracking/Tracker.cs

```diff
@@ -1120,6 +1125,21 @@ public override bool TrackItem(Item item, string? trackedAs = null, float? confi
         return didTrack;
     }
 
+    /// <summary>
+    /// Makes Tracker respond to a location if it was pre-configured by the user.
+    /// </summary>
+    /// <param name="location">The location at which an item was marked or tracked.</param>
+    /// <param name="marking"><see langword="true"/> if marking, <see langword="false"/> if tracking.</param>
+    private void GivePreConfiguredLocationSass(Location location, bool marking = false)
+    {
+        // "What a surprise."
+        if (LocalConfig != null && LocalConfig.LocationItems.ContainsKey(location.Id))
+        {
+            // I guess we're not storing the other options? We could respond to those, too, if we had those here.
```

Comment:
We do, it's under Config.ItemOptions. It's weird though because of how flexible it is (some different items are under the same group, some groups don't actually correspond to trackable items, etc.).

ItemSettingOptions has GetEarlyItemTypes that should get you what you want. Might need to be mindful to only say it once though per item in the case of stuff like an early sword.

Not sure why those are static classes rather than a service. 🤔 I'm guessing I had a reason for it, but it's long since been lost to me.

Reply:
Okay, that's something else. I was thinking about the general "progression/junk" thing but that's included, and since I don't actually check the value (only if the key is present), that should still work. 

The other way around (finding an item you picked to be early) would also be neat, but I'm not sure how to implement that. What's "early", exactly? Specific areas? Based on time? Based on amount of items picked up?

Maybe for another day.